app/dao/services_dao.py

```diff
@@ -72,6 +75,89 @@ def dao_count_live_services():
     ).count()
 
 
+def dao_fetch_live_services_data():
+    year_start_date, year_end_date = get_current_financial_year()
+    this_year_ft_billing = FactBilling.query.filter(
+        FactBilling.bst_date >= year_start_date,
+        FactBilling.bst_date <= year_end_date,
+    ).subquery()
+    data = db.session.query(
+        Service.id,
+        Organisation.name.label("organisation_name"),
+        Service.name.label("service_name"),
+        Service.consent_to_research,
+        Service.go_live_user_id,
+        User.name.label('user_name'),
+        User.email_address,
+        User.mobile_number,
+        Service.go_live_at.label("live_date"),
+        Service.volume_sms,
+        Service.volume_email,
+        Service.volume_letter,
+        case([
+            (this_year_ft_billing.c.notification_type == 'email', func.sum(this_year_ft_billing.c.notifications_sent))
+        ], else_=0).label("email_totals"),
+        case([
+            (this_year_ft_billing.c.notification_type == 'sms', func.sum(this_year_ft_billing.c.notifications_sent))
+        ], else_=0).label("sms_totals"),
+        case([
+            (this_year_ft_billing.c.notification_type == 'letter', func.sum(this_year_ft_billing.c.notifications_sent))
+        ], else_=0).label("letter_totals"),
+    ).outerjoin(
+        Service.organisation
```

Comment:
Do we need to filter by `bst_date`?

Reply:
We need to get totals for current financial year, so I think yes :)